include/uv-unix.h

```diff
@@ -137,8 +137,17 @@ typedef pthread_rwlock_t uv_rwlock_t;
 typedef UV_PLATFORM_SEM_T uv_sem_t;
 typedef pthread_cond_t uv_cond_t;
 typedef pthread_key_t uv_key_t;
-typedef pthread_barrier_t uv_barrier_t;
```

Comment:
this change should be part of the pthread-barrier compatibility shim. Check include/pthread-barrier.h and src/unix/pthread-barrier.c